docs/changes.txt

```diff
@@ -239,33 +239,215 @@ Changes in behaviour which may result in build errors
+- Allow using wxMessageQueue with move-only types (#25026).
+- Avoid DNS lookup when constructing wxIPV4address from IP (SMMB-DEV, #23109).
+- Fix multiple bugs in wxURI parsing (Ryan Norton, #23360).
+- Fix wxSocket::Peek() with UDP sockets (Brian Nixon, #23594).
+- Fix wxWebResponse::GetMimeType(), add GetContentType() (Blake-Madden, #23461).
+- Improve locale matching algorithm (Ulrich Telle, #24855)
+- Make wxString {To,From}CDouble() faster and more robust (#23287).
+- Make wxrc-generated code faster to build (DoctorNoobingstoneIPresume, #24579).
+- Multiple headers with same name in wxWebRequest (Stefan Dinkelacker, #24878).
+- Operators on wx types are not defined in global scope any more (#24190).
+- Optimize wxString by supporting move operations (Pavel Tyunin, #23215)
+- Optimize wxString::operator<<() for numeric types (#25388).
+- Preserve errno in wxString methods (Lauri Nurmi, #23113).
+- Support constructing wxString from std::string_view (Ian McInerney, #23711).
+- Update 3rd party libraries to latest versions (Maarten Bent, #25112, #25325).
+- Update Turkish translations (Kerim Demirkaynak, #25267).
+- Update language database (Ulrich Telle, #23437).
+- Use UTF-8 for environment variables (#25101).
+- Use variadic templates for vararg functions (#22981).
+- Use XDG-compliant location by default in wxFileConfig under Unix (#24195).
+
+All (GUI):
+
+- Add "feature" attribute to XRC (#23184).
+- Add RTF support to wxTextCtrl (Blake-Madden, #24626).
+- Add WXK_NUMPAD_CENTER (Ivan Sorokin, #23478).
+- Add XRC handler for wxVListBox (Bill Su, #24361).
+- Add checks for incorrect use of wxGridTable messages.
+- Add clearing browsing data in wxWebView (Tobias Taschner, #24431).
+- Add high DPI support to generic wxListCtrl (Gerhard Stein, #22916).
+- Add support for "id" attribute in wxHTML (Oleg Tolmatcev, #24968).
+- Add support for high DPI animations (#23817).
+- Add support for high DPI cursors via wxCursorBundle (#25374).
+- Add support for loading wxPropertyGrid from XRC (#24274).
+- Add support for printing multiple page ranges (Stefan Ziegler, #25030).
+- Add wxApp::SetAppearance() to explicitly set dark or light mode (#24461).
+- Add wxAuiManager::{Save,Load}Layout() (#24235).
+- Add wxGrid::CopySelection() (Blake Madden, #24124).
+- Add wxRadioButton group support to wxGenericValidator (Bill Su, #24264).
+- Add wxRibbonBar::GetPageById() (Blake Madden, #24211).
+- Add wxStatusBar::AddFieldControl() (AliKet, #24092).
+- Add wxTexTCtrl::SearchText() (Blake-Madden, #24756).
+- Add wxWebView advanced requests (Tobias Taschner, #22797).
+- Add wxWebView child window handling (Tobias Taschner, #23334).
+- Add wxWebView::EnableBrowserAcceleratorKeys() (Tobias Taschner, #23070)
+- Add wxWebView::EnablePersistentStorage() (Tobias Taschner, #24405).
+- Add wxWebView::GetNativeConfiguration() (Tobias Taschner, #22781).
+- Add wxWebView::SetProxy() (#23854).
+- Add wxWebViewConfiguration (Tobias Taschner, #23374).
```

Comment:
Speaking of wxWebView, in addition to new features listed above, there is also a new event wxEVT_WEBVIEW_CREATED (added in 92f6f16) and method ShowDevTools() (#24799). I believe that while these two were originally added as a part of Chromium integration, they are both supported at least by three backends (Chromium, Edge, WebKit2).